virttest/utils_netperf.py

```diff
@@ -124,8 +124,9 @@ def pack_compile(self, compile_option=""):
         build_type = {"aarch64": "aarch64-unknown-linux-gnu"}
         build_arch = self.session.cmd_output("arch", timeout=60).strip()
         np_build = build_type.get(build_arch, build_arch).strip()
-        setup_cmd = "./configure --build=%s %s > /dev/null 2>&1" % (np_build,
-                                                                    compile_option)
+        setup_cmd = "./autogen.sh > /dev/null 2>&1 &&" \
+                    " ./configure --build=%s %s > /dev/null 2>&1" % (np_build,
+                                                                     compile_option)
```

Comment:
I not sure about your change, so both versions have `autogen.sh` script? You will fail the setup if it does not exist?

Reply:
Yes,  The existing netperf version will release the 'autogen.sh' script after decompression.